test/BDNPerfTests/ConfigFiles/CI_CONFIG_BDN_Benchmark_Lua.LuaScripts.json

```diff
@@ -0,0 +1,19 @@
+{
+    "configuration": "Release",
```

Comment:
remove configuration (it will always be release), framework (it will always be .net8 - or change to .net9 when we are ready)